packages/web/validator-badges/README.md

```diff
@@ -22,6 +22,13 @@ Note:
 
 ## How to Claim Your Validator Badge
 
-1. Append your Validator or Valdiator Group address (starting with `0x`) as a new line to the corresponding markdown files in this current folder.
-2. Make a single pull request (PR) that contains all of your changes and is named `[Validator Badge] <your-validator-group-name>`.
-3. Validator badge PRs are reviewed once a week and once approved, your badges will be shown on the [Celo Validators Explorer](https://celo.org/validators/explore).
\ No newline at end of file
+1. If you are awarded with validator badge(s), check if your Validator Group address is present in the corresponding markdown files. If so, proceed to the next section; if not, proceed to step 2.
```

Comment:
I read this as if you are not awarded a badge add yourself - may be worth specifying "If you were awarded a badge and don't find your group address or the address is incorrect proceed to step 2"